k4FWCore/scripts/k4run

```diff
@@ -127,6 +127,11 @@ if __name__ == "__main__":
       print("usage: " + os.path.basename(__file__) + " [gaudi_config.py ...] Specify a gaudi options file to run")
       sys.exit()
 
+    opts_dict = vars(opts)
+    for optionName, propTuple in option_db.items():
+      print("Option name:", propTuple[1], optionName, opts_dict[optionName])
```

Comment:
When https://github.com/key4hep/k4FWCore/issues/50  is done, add this as `debug` output